demo-shell/src/app/services/app-notifications.service.ts

```diff
@@ -54,7 +54,7 @@ export class AppNotificationsService {
         private alfrescoApiService: AlfrescoApiService
     ) {
         this.alfrescoApiService.alfrescoApiInitialized.subscribe(() => {
-            if (this.isProcessServicesEnabled()) {
+            if (this.isProcessServicesEnabled() && this.areWSNotificationsEnabled()) {
```

Comment:
can we reduce technology from the naming and have it called `notificationsEnabled` function?